string object_id
string status
float32 time
